### PR TITLE
📖 Hack out old website contents, move new docs

### LINKED ISSUE #1535: Update kubestellar.io content (remove all non relevant documents from the docs folder)
1. Remove all documents related to our legacy code (example1, milestones, getting started, etc..)
2. Add a link to our updated 20.X.X content

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR does a quick-and-dirty job of deleting obsolete website contents and inserting pointers from the website to the documentation on GitHub. More work can be done in follow-on PRs.

This PR moves the current non-website docs from `docs/content/v0.20` to `docs/content/direct` because "0.20" is too specific.

## Related issue(s)

Fixes #1535

### CODE REVIEW

File: docs/content/readme.md [hunk 1]
```diff
@@ -40,25 +40,9 @@ __Don't change anything, just add KubeStellar!__
...
-
-
-## Roadmaps for the Project
-
-We have defined and largely completed the
-[PoC2023q1](../Coding%20Milestones/PoC2023q1/outline/).
-The current activity is refining the definition of, and producing, the
-[PoC2023q4](../Coding%20Milestones/PoC2023q4/outline/).
-Goals not addressed in that PoC are to be explored later.
+To find most of our documentation, go directly to the README of [the KubeStellar GitHub repository](http://github.com/kubestellar/kubestellar). To read the documentation for release `X.Y.Z`, view the Git content tagged `vX.Y.Z`.
```
Comment (ezrasilvera) [inline docs/content/readme.md:45]: This is not a "roadmap" statement.  I would remove this whole section.

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 ## Multi-cluster Configuration Management for Edge, Multi-Cloud, and Hybrid Cloud
 <br/>
 
-**NOTE**: the website kubestellar.io is temporarily out of order.
+**NOTE**: Some of our documentation is on the website [kubestellar.io](kubestellar.io), some is here.
```
Comment (ezrasilvera) [inline README.md:13]: Maybe:  
You can find more information on the project web site at  [kubestellar.io](kubestellar.io)  . Please note the site is still under construction.

File: docs/content/readme.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/content/readme.md:45]: Nothing alleges this sentence to be a roadmap statement. We need a pointer from here to the direct GitHub doc directory.

Comment (ezrasilvera) [reply docs/content/readme.md]: @MikeSpreitzer My bad.  I didn't see that it actually replace from line 45. Thanks

File: README.md (hunk 2 above)
Comment (MikeSpreitzer) [reply README.md:13]: I have reworded this.

**Comment by ezrasilvera** [comment]
Thanks.  Let's merge it as we are going to re-design and shuffle most of this in any case. 
/lgtm
/approve

**Pull request merged**